src/seam-connect/client.ts

```diff
@@ -143,7 +142,10 @@ export class Seam extends Routes {
 
     return makeRequest(client, {
       method: "POST",
```

Comment:
```suggestion
      method: "PUT",
```

Reply:
i've put PUT in here because in the fake implementation that is equivalent to `/client_sessions/get_or_create`

Reply:
we should make production match this behavior